README.md

```diff
@@ -178,10 +177,58 @@ FHIR resource or element containing primitive data types cannot be directly mapp
 
 To address this issue, the library generates
 [surrogate](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serializers.md#composite-serializer-via-surrogate)
-classes (e.g. `PatientSurrogate` in `Patient.kt`) for data classes containing primitive data types,
-mapping each primitive data type to two JSON properties . It also generates custom serializers (e.g.
-`PatienetSerializer` in `Patient.kt`) that delegate the serialization/deserialization process to the
-corresponding surrogate classes and translate between the data classes and surrogate classes.
+classes (e.g. `PatientSurrogate`) for data classes containing primitive data types, mapping each
+primitive data type to two JSON properties . It also generates custom serializers (e.g.
+`PatienetSerializer`) that delegate the serialization/deserialization process to the corresponding
```

Comment:
```suggestion
`PatientSerializer`) that delegates the serialization/deserialization process to the corresponding
```

Reply:
thanks for the review elly - i'll fix the spelling of the patient serializer, but it should be delegate since it's corresponding to "custom serializers" before the paranthesis.